actionpack/test/dispatch/host_authorization_test.rb

```diff
@@ -152,7 +152,7 @@ class HostAuthorizationTest < ActionDispatch::IntegrationTest
     }
 
     assert_response :forbidden
-    assert_match "Blocked host: 127.0.0.1", response.body
+    assert_match "Blocked host: www.example.com", response.body
```

Comment:
@gsamokovarov, can you please take a quick look at those changes in tests? I described my reason for those changes in https://github.com/rails/rails/pull/42135/commits/b05ce1c9cb1a18bf1ee4aa1698f3e33da54e24b1.